rust-runtime/aws-smithy-runtime/src/client/http/body/minimum_throughput/options.rs

```diff
@@ -136,19 +129,26 @@ impl MinimumThroughputBodyOptionsBuilder {
         self
     }
 
-    /// Set the rate at which throughput is checked.
-    ///
-    /// Defaults to 1 second.
-    pub fn check_interval(mut self, check_interval: Duration) -> Self {
-        self.set_check_interval(Some(check_interval));
+    /// No longer used.
+    #[deprecated(note = "No longer used.")]
+    pub fn check_interval(self, _check_interval: Duration) -> Self {
         self
     }
 
-    /// Set the rate at which throughput is checked.
-    ///
-    /// Defaults to 1 second.
-    pub fn set_check_interval(&mut self, check_interval: Option<Duration>) -> &mut Self {
-        self.check_interval = check_interval;
+    /// No longer used.
```

Comment:
deprecation should probably refer to `check_window` since that appears to be the replacement?

Reply:
I'm not sure I want to make check_window actually configurable or not yet. I was leaning towards not exposing it until a use-case came up for it.

Reply:
Offline discussion: Improve deprecation messaging to indicate that this configuration is not currently available and to file an issue if it's needed.